pkg/csi_driver/utils.go

```diff
@@ -457,6 +458,9 @@ func checkGcsFuseErr(isInitContainer bool, pod *corev1.Pod, targetPath string) (
 			return codes.Unavailable, fmt.Errorf("benign grpc error: %s", errMsgStr)
 		}
 
+		if deprecatedFlagPatterns.MatchString(errMsgStr) {
```

Comment:
I know you were just  following pattern of everyone else, but it actually would be good to have a unit test for this. Can you please move everything starting from

 	
```
if err == nil && len(errMsg) > 0 {
```

into a helper function parseCodeFromGCSFuseError or something similar. Have it return an error code + the wrapped error. Then you can unit test it to be sure the exact string we saw in SLO will be fixed with this.